woocommerce/woocommerce
# [REST API] Reports totals endpoints

### All Submissions:






### Changes proposed in this Pull Request:



Introduces new 5 endpoints showing total of orders per status, customers (paying or not), coupons per type, products per type and reviews per rating.

Closes # 21115.

### How to test the changes in this Pull Request:

1. GET `/wp-json/wc/v3/reports/orders/totals` and check output
2. GET `/wp-json/wc/v3/reports/products/totals` and check output
3. GET `/wp-json/wc/v3/reports/coupons/totals` and check output
4. GET `/wp-json/wc/v3/reports/customers/totals` and check output
5. GET `/wp-json/wc/v3/reports/reviews/totals` and check output

### Other information:




### Changelog entry

> Dev - REST API - Included new reports endpoints to display total of orders, products, customers, coupons and reviews.
